PC Creator 2 - Computer Tycoon (#299)
riverar/wsa-app-compatibility: README.md

| New Star Soccer | 4.27 | 13, 12, 11 | ✅ | Keyboard not supported
| osu!lazer | 2023.403.1 | 13 - 5 | ⚠️ | Runs with terrible performance, high latency, generally unplayable
| PAC-MAN (Play Games) | 2023.08.46243 | 13 | 🆖 || Requires GMS, keyboard supported

> | PC Creator 2 - Computer Tycoon | 4.1.5 | 13 | ❌ |Mouse does not work with the tutorial so  you can not complete it thus being stuck| the game starts fine but you can not move the tutorial pc to install windows so the game does not work|

Reviewer: ```suggestion
| PC Creator 2 - Computer Tycoon | 4.1.5 | 13 | ❌ | Mouse does not work with the tutorial, so it cannot be completed ||
```
